lte/cloud/go/services/lte/servicers/builder_servicer_test.go

```diff
@@ -35,6 +40,173 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestBuilder_Build_BaseCase(t *testing.T) {
```

Comment:
I just refactored this test so i can reuse its functions on the new `TestBuilder_Build_FederatedAGW`